site/src/pages/components/interest-invokers.explainer.mdx

```diff
@@ -247,16 +259,76 @@ Using those insets, a developer could implement a long-press activated hovercard
+
+The same is not true for `interesttarget`. When the target is a popover, the only set of actions that makes sense is to *show* the popover when interest is shown, and *hide* the popover when interest is lost. There are no identified use cases that invert those actions. Alternatively, defining show and lose interest actions to both *toggle* the popover can lead to serious out-of-sync problems that would be major footguns. So for popovers, there is no need for a separate attribute controlling the behavior, because there is only one rational set of behaviors. And as described above, popovers are the only kind of target element that comes with default actions. Other actions might use the `"interest"` and `"loseinterest"` events to add additional behavior, but in that case, they can provide their own definitions of actions to take. And again because of the above points, there isn't a future-compat problem like there might be for the `commandfor` attribute.
+
+See https://github.com/openui/open-ui/issues/1064#issuecomment-2581511411 (particularly [this comment](https://github.com/openui/open-ui/issues/1064#issuecomment-2581511411)) for more discussion of this section.
+
+### Why a keyboard hot-key and not just keyboard focus?
+
+There are pros and cons to both methods.
+
+#### Focus element to show interest (possibly after delay)
+
+There are two approaches: instantly show interest when an element is focused, or show interest after the element has been focused for a period of time. Most people seem to favor a delay, to avoid extreme annoyance for keyboard users. Perhaps re-use [the mouse delays](#Mouse-delays) or create a separate version for keyboard focus delays. To avoid accidental neglect of the keyboard delays/users, it might be better to re-use the same delays for both mouse and keyboard.
+
+Pros:
+- very discoverable - simply focus an element and wait.
+
+Cons:
+- hard to avoid / annoying. Users simply trying to keyboard-navigate through a document might inadvertently trigger popovers. Note that if the target popover is a "rich hint", it will be next in the tab navigation cycle, forcing the user on a "detour" in this case. It's further possible that the rich hint itself has additional interest targets, which can open their own rich hints, trapping the user for a significant number of tab stops.
+- can be slow to activate, due to the delay.
+
+#### Hot-key to show interest
+
+Pros:
+- No chance for "hijacking" the tab navigation cycle.
+- Fast/immediate to activate (no delays needed).
+
+Cons:
+- Not very discoverable.
+
+#### Conclusion: hot-key
```

Comment:
If I had my druthers the conclusion would be try both and let everyone play with it until we figure out which is best, but I know this is potentially more development cycles. It might be fewer dev cycles in the end though, because it will settle the argument better if we tried both and there was a clear consensus.

Reply:
I think in most cases, people who are tabbing are pressing tab pretty rapidly. Admittedly this falls apart if the person is unable to press keys quickly. However, I think finding a good setting for the dwell-before-show time is probably possible, where it doesn't lead to lots of extra popovers showing, doesn't lead to extra tabbing, and is more discoverable than Alt+down.

Reply:
I think that's the point of this explainer and the conversation (tomorrow) about this. I'd rather not implement *both*, if we can resolve it ahead of time.

Reply:
Maybe we implement Alt+down/space/whatever, and then a javascript bookmarklet to open on a timer, that people can tweak/play with as a prototype to what it would be like. Just so that we can show that it actually is or isn't a good idea.